__tests__/app.test.js

```diff
@@ -113,11 +113,9 @@ describe("GET /api/articles", () => {
       .expect(200)
       .then(({ body: { articles } }) => {
         expect(articles.length).toBeGreaterThan(0);
-        //cannot add "jest": {"setupFilesAfterEnv": ["jest-sorted"] to package.json without messing up the util-test and jest-extended/all dependency
-        //}
-        // expect(articles).toBeSortedBy("created_at", {
-        //   descending: true,
-        // });
```

Comment:
You shouldn't need to add that to the package.json - just requiring in jest sorted at the top of the file should be enough to get your tests with .toBeSorted by to work